handbook/product/README.md

```diff
@@ -77,7 +77,9 @@ For external contributors: please consider opening an issue with reference scree
 
 Once the draft has been approved, it moves to the "Settled" column on the drafting board. 
 
-Before assigning an engineering manager for [estimation](https://fleetdm.com/handbook/engineering#sprint-ceremonies), the product team should ensure the product section of the user story [checklist](https://github.com/fleetdm/fleet/issues/new?assignees=&labels=story&projects=&template=story.md&title=) is complete. 
+Before assigning an engineering manager to [estimate](https://fleetdm.com/handbook/engineering#sprint-ceremonies) a user story, the product team should ensure the product section of the user story [checklist](https://github.com/fleetdm/fleet/issues/new?assignees=&labels=story&projects=&template=story.md&title=) is complete. 
+
+For bugs that have gone through design and are considered "Settled", the designer should remove the `:product` label and move the issue to the 'To be scheduled' column on the bugs board for the attention of the product manager.
```

Comment:
```suggestion
Once a bug has gone through design and is considered "Settled", the designer removes the `:product` label and moves the issue to the 'To be scheduled' column on the "Bugs" board for the attention of the product manager.
```

What prevents this from going directly to the EM?

Reply:
Unrelated to Mike's question above: When does a bug need to go through design?

Reply:
@mikermcneil the idea is for the bug to be routed through the PM first so it can be appropriately prioritized into the sprint backlog. This aims to avoid things slipping through or being prioritized incorrectly. I've updated the language to be more specific about this, so thanks for flagging that it wasn't clear.